src/java/org/apache/cassandra/utils/NoSpamLogger.java

```diff
@@ -51,19 +51,18 @@ public enum Level
     }
 
     @VisibleForTesting
-    static interface Clock
+    public interface Clock
```

Comment:
this was done in https://issues.apache.org/jira/browse/CASSANDRA-18952; simulation detects that global clock is used and rejects it, so we now use the simulated clock here, which means the no spam logic happens often (simulation time moves much faster than wall clock)